Signal/src/call/OutboundCallInitiator.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
     public func initiateCall(handle: String) -> Bool {
         Logger.info("\(TAG) in \(#function) with handle: \(handle)")
 
-        guard let recipientId = PhoneNumber(fromUserSpecifiedText: handle)?.toE164() else {
```

Comment:
I believe this could be user specified text when calling from the contacts app. We could push that parsing farther up the chain though.

It would be nice to use a type system to distinguish between "a bunch of text" and "a valid signal identitifer"